Modules/NameColorManager.cs

```diff
@@ -177,8 +181,10 @@ private static bool KnowTargetRoleColor(PlayerControl seer, PlayerControl target
             || (target.Is(CustomRoles.Mayor) && Options.MayorRevealWhenDoneTasks.GetBool() && target.GetPlayerTaskState().IsTaskFinished)
             || (seer.Is(CustomRoleTypes.Crewmate) && target.Is(CustomRoles.Marshall) && target.GetPlayerTaskState().IsTaskFinished)
          //   || Mare.KnowTargetRoleColor(target, isMeeting)
-            || EvilDiviner.IsShowTargetRole(seer, target)
-            || PotionMaster.IsShowTargetRole(seer, target);
+            || (target.Is(CustomRoles.NiceMini) && Mini.EveryoneCanKnowMini.GetBool())

```

Comment:
Why **|| EvilDiviner.IsShowTargetRole(seer, target) || PotionMaster.IsShowTargetRole(seer, target)** was deleted, lol

Reply:
On it

Reply:
Fixed